### ISSUE TITLE
keys * can't work with huge of DB

### ISSUE THREAD

**Issue by huacnlee** [yankov/redis-migrator#6]
I had tried use redis-migrator to split a large DB, there have more than 20,000,000 keys in Redis.

In [this line](https://github.com/yankov/redis-migrator/blob/master/lib/redis_migrator.rb#L30), I saw that use `keys(*)` to get all of the keys from Redis, that can't work, because it will timeout.

**Comment by yankov** [comment]
That's true. Keys \* was not a good idea. Now redis supports "scan" which allows to iterate through the set of keys. We need to use it instead. http://redis.io/commands/scan
